styleguide.md

```diff
@@ -0,0 +1,180 @@
+# Modelica Specification style guide
+
+This is the style guide for the Modelica Specification document.
+
+
+## Document format and tool chain
+
+The source format of the document is LaTeX, and the source is processed by both pdfLaTeX and LaTeXML.
+It is good to be aware of the LaTeXML implications, but for most contributors and contributions is is considered sufficient to only check that the pdfLaTeX output looks good, the idea being that potential problems should be spotted in the pull request review process.
+
```

Comment:
I believe we already here should add that _there should be no trailing spaces_ (as that is part of the checks in the tool-chain).

Reply:
Very good suggestion!  Please have a look add added rules for trailing space and empty lines.

Reply:
Based on verbal communication with @HansOlsson, I think that this should be considered resolved.